appserver/tests/payara-samples/samples/asadmin/src/test/java/fish/payara/samples/asadmin/AutoNameInstancesTest.java

```diff
@@ -51,7 +51,7 @@
 @NotMicroCompatible
 public class AutoNameInstancesTest extends AsadminTest {
 
-    private static String domainName = "domain1";
+    private static String domainName = "test-domain";
```

Comment:
😕
Not a fan, even as a temporary workaround - this completely shafts the managed profiles.

Reply:
true, but remote if left as is

Reply:
Could you not change it to check for a system property as a proper workaround?
This is _only_ fixing it for our internal CI - nobody is making a domain called test-domain when testing manually, so it arguably breaks both remote and managed profiles.

Either that or just disable the test temporarily - I think this is a bad workaround.